_tools/check_unstable_deps.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+// deno-lint-ignore-file no-console
+
+/**
+ * This script checks if the stable modules import the unstable modules.
+ *
+ * Usage: `deno run -A _tools/check_unstable_deps.ts`
+ *
+ * @module
+ */
+
+// These 2 paths are known to include unstable module (net/unstable_get_network_address.ts)
+// and should be ignored.
+const EXCEPTOIONS = [
+  "std/http/file_server.ts",
+  "std/http/mod.ts",
+];
+
+import denoJson from "../deno.json" with { type: "json" };
+import { join } from "@std/path/unstable-join";
+import { greaterOrEqual, parse } from "@std/semver";
+import { zip } from "@std/collections/zip";
+import { resolveWorkspaceSpecifiers } from "./utils.ts";
+import { createGraph } from "@deno/graph";
+
+type DenoJson = { version: string; exports: Record<string, string> };
+function readDenoJson(path: string): Promise<DenoJson> {
+  return Deno.readTextFile(join(path, "deno.json")).then(JSON.parse);
+}
```

Comment:
You could also just dynamically `import`.

Reply:
The code without dynamic import looks more concise to me:
```diff
-return Deno.readTextFile(join(path, "deno.json")).then(JSON.parse);
+return (await import(`../${path}/deno.json`, { with: { type: "json" } })).default;
```